Makefile

```diff
@@ -7,7 +7,7 @@ venv: $(VENV_NAME)/bin/activate
 $(VENV_NAME)/bin/activate: setup.py
 	$(PIP) install --upgrade pip virtualenv
 	@test -d $(VENV_NAME) || $(PYTHON) -m virtualenv --clear $(VENV_NAME)
-	${VENV_NAME}/bin/python -m pip install -U pip tox twine
+	${VENV_NAME}/bin/python -m pip install -U pip tox twine "cryptography<40"
```

Comment:
can we move these to `requirement.txt` file as [seems to be customary](https://grep.app/search?q=cryptography%3C&filter[path.pattern][0]=requirements.txt) for python?